### PR TITLE
Added scrollIntoView() and focus() clarifications

### PR DESCRIPTION
**Pull request by vmpstr**
Please take a look.

This one has a bunch of links, so for now you can see the rendered output here:
https://vmpstr.github.io/display-locking/#restrictions

I also numbered the clarifications and restrictions to make it easier to refer to.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -252,6 +252,38 @@ Restrictions and Clarifications {#restrictions}
       update) will retrieve values consistent with current painted state and not cause any forced layouts.
     </div>
 
+6. For an [=off-screen=] element with ''subtree-visibility: auto'' and for
+    elements in its subtree, <a
+    href="https://drafts.csswg.org/cssom-view/#dom-element-scrollintoview"><code>scrollIntoView()</code></a>
+    computes the element's bounds considering [=size containment=] that is applied
+    on the ''subtree-visibility: auto'' element.
+    <div class=note>
+      <code>scrollIntoView()</code> brings the targetted element into the
```
Comment (chrishtr) [inline index.bs]:
```suggestion
      <code>scrollIntoView()</code> brings the targeted element into the
```

File: index.bs [hunk 2]
```diff
@@ -252,6 +252,38 @@ Restrictions and Clarifications {#restrictions}
       update) will retrieve values consistent with current painted state and not cause any forced layouts.
     </div>
 
+6. For an [=off-screen=] element with ''subtree-visibility: auto'' and for
+    elements in its subtree, <a
+    href="https://drafts.csswg.org/cssom-view/#dom-element-scrollintoview"><code>scrollIntoView()</code></a>
+    computes the element's bounds considering [=size containment=] that is applied
```
Comment (chrishtr) [inline index.bs]:
```suggestion
    computes the element's bounds with its existing [=size containment=] applied
```

File: index.bs [hunk 3]
```diff
@@ -252,6 +252,38 @@ Restrictions and Clarifications {#restrictions}
...
+    on the ''subtree-visibility: auto'' element.
+    <div class=note>
+      <code>scrollIntoView()</code> brings the targetted element into the
+      viewport. This means that elements with ''subtree-visibility: auto'' will
+      not be [=skipped=] and thus will not have [=size containment=]
+      automatically applied to them. However, since
```
Comment (chrishtr) [inline index.bs]:
```suggestion
      automatically applied to them after scroll is applied. However, since
```

**Comment by vmpstr** [comment]
PTAL

**Pull request merged**